package.json

```diff
@@ -38,7 +38,8 @@
         "lint:check": "eslint ./src ./tests --ext .ts,.tsx",
         "format:write": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
         "format:check": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\"",
-        "release": "release-it"
+        "release": "release-it",
+        "typecheck": "tsc --noEmit"
```

Comment:
should I revert this? used it for testing...

Reply:
Seems useful to leave it there 👍